src/MICore/LaunchOptions.cs

```diff
@@ -53,9 +53,14 @@ public enum LaunchCompleteCommand
         ExecContinue,
 
         /// <summary>
+        /// Starts the debugger in the stopped state for debugging a core dump.
+        /// </summary>
+        CoreDump,
```

Comment:
I'm not sure this is the right place for this. The launch complete command is designed for users to manually insert gdb commands into the launch sequence. I would expect the core dump support to be a first class citizen and detected by the presense of a coredump element or attribute in the local launch options.